Makefile

```diff
@@ -14,9 +14,9 @@ check-version:
 		echo "Version check passed" ; \
 	fi
 
-.PHONY: wheel
+.PHONY: build
 wheel:
-	poetry build -f wheel
+	poetry build
```

Comment:
I have a concern with this, reason being is that freeze-wheel only freezes the .whl package, not the sdist. So anyone using sdist can have a dependency issue @danut13